src/components/Checkbox/Checkbox.module.css

```diff
@@ -14,6 +14,7 @@
 }
 
 .checkIconContainer {
+  display: inline-block;
```

Comment:
The checkbox was an inline element. They were extending vertically depending on their placement.